Makefile

```diff
@@ -73,6 +73,7 @@ str_match_test:
 # testing
 ##
 test: joy joy_test.py
+	./download_test_pcaps.sh test_pcaps/pcapr
```

Comment:
Good point--I'll double check on this. Do you think it makes sense to download these pcaps in the first place? I can also just keep this as part of testing on my devel branch without merging into master.